### ISSUE TITLE
Proposal: Change the format of build numbers to YYYY-MM-DD.NN

### ISSUE THREAD

**Issue by erikmd** [deajan/osync#25]
Hello,

This is really a detail, but it is maybe better to deal with this minor issue before the pending release:

The fact is that the timestamp format currently used is "DDMMYYYYNN", which is not that readable, and does not yield increasing build numbers.
So I would suggest to use something like "YYYY-MM-DD.NN" (that is, using ISO 8601 for the date), so the corresponding strings would be increasing with respect to the lexicographic order.

To sum up,
```bash
PROGRAM_BUILD=2606201501
```
would become
```bash
PROGRAM_BUILD=2015-06-26.01
```
Do you see any inconvenient to perform such a change?

(By the way, the date itself can be obtained in this format by using the `date -I` or `date +%F` shell command.)

Best regards.

**Comment by deajan** [comment]
PROGRAM_BUILD won't interfere with anything, so this ain't an issue.
It's more a variable to know exactly which version i'm working on, not used anywhere alse.
I am working in France and adopted the French dating system.

Adopting an international dating system might be a better idea indeed.
What about keeping it simple and use the same serial number system as bind servers ?
PROGRAM_BUILD=YYYYMMDDNN

**Comment by erikmd** [comment]
Hello,

> Adopting an international dating system might be a better idea indeed.
> What about keeping it simple and use the same serial number system as bind servers ?
> PROGRAM_BUILD=YYYYMMDDNN

If `PROGRAM_BUILD=YYYYMMDDNN` is OK for you, I see no objection for
choosing this simpler format. Indeed beyond the fact that it is
"international", it still has the nice property of being an increasing
number over time…

**Comment by deajan** [comment]
YYYYMMDDNN it is !
